libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -205,6 +205,9 @@ MAV_MOUNT_MODE AP_Mount::get_mode(uint8_t instance) const
 //      this operation requires 60us on a Pixhawk/PX4
 void AP_Mount::set_mode_to_default(uint8_t instance)
 {
+    if (instance > ARRAY_SIZE(_params)) {
```

Comment:
we've got an instance check that could be used right?  or maybe AP_MOUNT_MAX_INSTANCES?

Reply:
Yeah, I tried both ways in here.  Considering what we're dereferencing on the next few lines, I went this way eventually.  There's actually another method in here which checks against AP_MOUNT_MAX_INSTANCES (rather than call check_isntances) - but right here checking the array size seems like the right thing to do to me....